.travis.yml

```
@@ -98,31 +98,30 @@ jobs:
         - skip
     - stage: conda-build
       install:
-        - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
-        - bash miniconda.sh -b -p $HOME/miniconda
-        - export PATH="$HOME/miniconda/bin:$PATH"
+        - cd ../
+        - bash ./tethys/scripts/install_tethys.sh --partial-tethys-install me -n tethys -s $PWD/tethys
+        - . ~/miniconda/etc/profile.d/conda.sh
+        - conda activate tethys
         - hash -r
         - conda config --set always_yes yes --set changeps1 no
         - conda update -q conda
         - conda install conda-build anaconda-client
-        - conda config --add channels bokeh/label/dev
-        - conda config --add channels conda-forge
-        - conda config --add channels tethysplatform/label/dev
-        - conda config --add channels tethysplatform
         - conda config --set anaconda_upload no
         - mkdir -p ~/conda-bld
       script:
         - echo "Building conda package"
         - export CONDA_BLD_PATH=~/conda-bld
-        - conda build ./conda.recipe
+        - conda build ./tethys/conda.recipe
```

```suggestion
        - conda build -c tethysplatform/label/dev -c conda-forge ./tethys/conda.recipe
```

The `tethysplatform/label/dev` channel is needed for now because I haven't moved all of the dependencies over to `main`. I'll see if I can do that before next week.

---

@sdc50 I added a commit with this pointing to ``tethysplatform`` rather than ``tethysplatform/label/dev`` because it seemed you had moved everything over.